account-kit/wallet-client/src/client/actions/sendCalls.ts

```diff
@@ -0,0 +1,69 @@
+import type { Address } from "viem";
+import type { InnerWalletApiClient } from "../../types.js";
+import { prepareCalls, type PrepareCallsParams } from "./prepareCalls.js";
+import { metrics } from "../../metrics.js";
+import { signPreparedCalls } from "./signPreparedCalls.js";
+import type { SmartAccountSigner } from "@aa-sdk/core";
+import {
+  sendPreparedCalls,
+  type SendPreparedCallsResult,
+} from "./sendPreparedCalls.js";
+
+export type SendCallsParams<
+  TAccount extends Address | undefined = Address | undefined,
+> = PrepareCallsParams<TAccount>;
+
+export type SendCallsResult = SendPreparedCallsResult;
+
+/**
+ * Prepares, signs, and submits calls. This function internally calls `prepareCalls`, `signPreparedCalls`, and `sendPreparedCalls`.
+ *
+ * @param {InnerWalletApiClient} client - The wallet API client to use for the request
+ * @param {SmartAccountSigner} signer - The signer to use
+ * @param {PrepareCallsParams<TAccount>} params - Parameters for sending calls
+ * @param {Array<{to: Address, data?: Hex, value?: Hex}>} params.calls - Array of contract calls to execute
+ * @param {Address} [params.from] - The address to execute the calls from (required if the client wasn't initialized with an account)
+ * @param {object} [params.capabilities] - Optional capabilities to include with the request.
+ * @returns {Promise<SendPreparedCallsResult>} A Promise that resolves to the result containing the prepared call IDs.
+ *
+ *  @example
+ * ```ts
+ * // First prepare the calls
```

Comment:
```suggestion
 * // Send the calls
```
(or maybe just remove this comment)